### DISCUSSION TITLE
Container boot loop

### DISCUSSION THREAD

**Discussion by finnaGIT** [open-webui/open-webui#7140] [Issues]
# Bug Report

## Installation Method

Docker command line, on Ubuntu 22.04 via WSL2 on windows 11

## Environment

- **Open WebUI Version:** v0.4.x (latest ‘:dev-cuda’ as of 11/21/2024.
- **Ollama (if applicable):** n/a

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]

**Confirmation:**

- [X ] I have read and followed all the instructions provided in the README.md.
- [X ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [X ] I have included the Docker container logs.
- [ ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Expect the web server to start

## Actual Behavior:

Web server inaccessible. Docker container logs in continuous loop of items shown below. 

## Description

**Bug Summary:**
I added the Nvidia/nvidia-embed-v2 into the embed model then saw logs that said I needed to set `RAG_EMBEDDING_MODEL_TRUST_REMOTE_CODE

`=true. I shutdown the container, changed that value from false to true, pulled the latest image, and encountered this error/bug.

## Reproduction Details

**Steps to Reproduce:**
unknown how to reproduce but the steps above are what happened

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
‘’’
`INFO  [open_webui.env] 'IMAGE_GENERATION_MODEL' loaded from the latest database entry
INFO  [open_webui.apps.audio.main] whisper_device_type: cuda
WARNI [langchain_community.utils.user_agent] USER_AGENT environment variable not set, consider setting it to identify your requests.

Fetching 19 files:   0%|          | 0/19 [00:00<?, ?it/s]
Fetching 19 files: 100%|██████████| 19/19 [00:00<00:00, 97066.72it/s]

Loading checkpoint shards:   0%|          | 0/4 [00:00<?, ?it/s]
Loading checkpoint shards:  25%|██▌       | 1/4 [00:01<00:03,  1.12s/it]
Loading checkpoint shards:  50%|█████     | 2/4 [00:02<00:02,  1.15s/it]MLoading WEBUI_SECRET_KEY from file, not provided as an environment variable.
Loading WEBUI_SECRET_KEY from .webui_secret_key
CUDA is enabled, appending LD_LIBRARY_PATH to include torch/cudnn & cublas libraries.
/app/backend/open_webui

/app/backend
/app
Running migrations
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [open_webui.env] 'ENABLE_API_KEY' loaded from the latest database entry
INFO  [open_webui.env] 'JWT_EXPIRES_IN' loaded from the latest database entry
INFO  [open_webui.env] 'ENABLE_OLLAMA_API' loaded from the latest database entry
INFO  [open_webui.env] 'OLLAMA_BASE_URLS' loaded from the latest database entry
INFO  [open_webui.env] 'OLLAMA_API_CONFIGS' loaded from the latest database entry
INFO  [open_webui.env] 'ENABLE_OPENAI_API' loaded from the latest database entry
INFO  [open_webui.env] 'OPENAI_API_KEYS' loaded from the latest database entry
INFO  [open_webui.env] 'OPENAI_API_BASE_URLS' loaded from the latest database entry
INFO  [open_webui.env] 'OPENAI_API_CONFIGS' loaded from the latest database entry
INFO  [open_webui.env] 'ENABLE_SIGNUP' loaded from the latest database entry
INFO  [open_webui.env] 'DEFAULT_LOCALE' loaded from the latest database entry
INFO  [open_webui.env] 'DEFAULT_PROMPT_SUGGESTIONS' loaded from the latest database entry
INFO  [open_webui.env] 'DEFAULT_USER_ROLE' loaded from the latest database entry
INFO  [open_webui.env] 'ENABLE_EVALUATION_ARENA_MODELS' loaded from the latest database entry
INFO  [open_webui.env] 'EVALUATION_ARENA_MODELS' loaded from the latest database entry
INFO  [open_webui.env] 'WEBHOOK_URL' loaded from the latest database entry
INFO  [open_webui.env] 'ENABLE_COMMUNITY_SHARING' loaded from the latest database entry
INFO  [open_webui.env] 'ENABLE_MESSAGE_RATING' loaded from the latest database entry
WARNI [open_webui.env] 
WARNING: CORS_ALLOW_ORIGIN IS SET TO '*' - NOT RECOMMENDED FOR PRODUCTION DEPLOYMENTS.
INFO  [open_webui.env] 'SHOW_ADMIN_DETAILS' loaded from the latest database entry
INFO  [open_webui.env] 'CONTENT_EXTRACTION_ENGINE' loaded from the latest database entry
INFO  [open_webui.env] 'TIKA_SERVER_URL' loaded from the latest database entry
INFO  [open_webui.env] 'RAG_TOP_K' loaded from the latest database entry
INFO  [open_webui.env] 'RAG_RELEVANCE_THRESHOLD' loaded from the latest database entry
INFO  [open_webui.env] 'ENABLE_RAG_HYBRID_SEARCH' loaded from the latest database entry
INFO  [open_webui.env] 'ENABLE_RAG_WEB_LOADER_SSL_VERIFICATION' loaded from the latest database entry
INFO  [open_webui.env] 'RAG_EMBEDDING_ENGINE' loaded from the latest database entry
INFO  [open_webui.env] 'PDF_EXTRACT_IMAGES' loaded from the latest database entry
INFO  [open_webui.env] 'RAG_EMBEDDING_MODEL' loaded from the latest database entry
INFO  [open_webui.env] Embedding model set: nvidia/NV-Embed-v2
INFO  [open_webui.env] 'RAG_RERANKING_MODEL' loaded from the latest database entry
INFO  [open_webui.env] Reranking model set: BAAI/bge-reranker-v2-m3
INFO  [open_webui.env] 'RAG_TEXT_SPLITTER' loaded from the latest database entry
INFO  [open_webui.env] 'CHUNK_SIZE' loaded from the latest database entry
INFO  [open_webui.env] 'CHUNK_OVERLAP' loaded from the latest database entry
INFO  [open_webui.env] 'RAG_TEMPLATE' loaded from the latest database entry
INFO  [open_webui.env] 'YOUTUBE_LOADER_LANGUAGE' loaded from the latest database entry
INFO  [open_webui.env] 'ENABLE_RAG_WEB_SEARCH' loaded from the latest database entry
INFO  [open_webui.env] 'RAG_WEB_SEARCH_ENGINE' loaded from the latest database entry
INFO  [open_webui.env] 'SEARXNG_QUERY_URL' loaded from the latest database entry
INFO  [open_webui.env] 'GOOGLE_PSE_API_KEY' loaded from the latest database entry
INFO  [open_webui.env] 'GOOGLE_PSE_ENGINE_ID' loaded from the latest database entry
INFO  [open_webui.env] 'BRAVE_SEARCH_API_KEY' loaded from the latest database entry
INFO  [open_webui.env] 'SERPSTACK_API_KEY' loaded from the latest database entry
INFO  [open_webui.env] 'SERPSTACK_HTTPS' loaded from the latest database entry
INFO  [open_webui.env] 'SERPER_API_KEY' loaded from the latest database entry
INFO  [open_webui.env] 'SERPLY_API_KEY' loaded from the latest database entry
INFO  [open_webui.env] 'TAVILY_API_KEY' loaded from the latest database entry
INFO  [open_webui.env] 'JINA_API_KEY' loaded from the latest database entry
INFO  [open_webui.env] 'SEARCHAPI_API_KEY' loaded from the latest database entry
INFO  [open_webui.env] 'BING_SEARCH_V7_ENDPOINT' loaded from the latest database entry
INFO  [open_webui.env] 'BING_SEARCH_V7_SUBSCRIPTION_KEY' loaded from the latest database entry
INFO  [open_webui.env] 'RAG_WEB_SEARCH_RESULT_COUNT' loaded from the latest database entry
INFO  [open_webui.env] 'RAG_WEB_SEARCH_CONCURRENT_REQUESTS' loaded from the latest database entry
INFO  [open_webui.env] 'IMAGE_GENERATION_ENGINE' loaded from the latest database entry
INFO  [open_webui.env] 'ENABLE_IMAGE_GENERATION' loaded from the latest database entry
INFO  [open_webui.env] 'AUTOMATIC1111_BASE_URL' loaded from the latest database entry
INFO  [open_webui.env] 'AUTOMATIC1111_API_AUTH' loaded from the latest database entry
INFO  [open_webui.env] 'AUTOMATIC1111_CFG_SCALE' loaded from the latest database entry
INFO  [open_webui.env] 'AUTOMATIC1111_SAMPLER' loaded from the latest database entry
INFO  [open_webui.env] 'AUTOMATIC1111_SCHEDULER' loaded from the latest database entry
INFO  [open_webui.env] 'COMFYUI_BASE_URL' loaded from the latest database entry
INFO  [open_webui.env] 'COMFYUI_WORKFLOW' loaded from the latest database entry
INFO  [open_webui.env] 'COMFYUI_WORKFLOW' loaded from the latest database entry
INFO  [open_webui.env] 'IMAGES_OPENAI_API_BASE_URL' loaded from the latest database entry
INFO  [open_webui.env] 'IMAGES_OPENAI_API_KEY' loaded from the latest database entry
INFO  [open_webui.env] 'IMAGE_SIZE' loaded from the latest database entry
INFO  [open_webui.env] 'IMAGE_STEPS' loaded from the latest database entry
INFO  [open_webui.env] 'IMAGE_GENERATION_MODEL' loaded from the latest database entry`

**Comment by finnaGIT** [comment]
Hi, kinda new around here. This was marked as a discussion. What happens next? Thanks!

**Comment by fdkgenie** [comment]
I got the same problem when use Embedding model: nvidia/NV-Embed-v2
